futures-async-stream-macro/src/stream.rs

```diff
@@ -119,16 +118,16 @@ impl Parse for Args {
     }
 }
 
-struct FnSig {
-    attrs: Vec<Attribute>,
-    vis: Visibility,
-    sig: Signature,
-    block: Block,
-    semi: Option<Token![;]>,
+pub(super) struct FnSig {
+    pub(super) attrs: Vec<Attribute>,
```

Comment:
<nitpicking>I think it's nicer to use `pub` instead of `pub(...)` for struct fields, since the struct's own visibility inherently creates an "upper bound" on the fields' visibility.</nitpicking>

Reply:
I have no strong opinion about it, but [`unreachable_pub` warns that.](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=e544d6d3b71494f65337b1e4137fd2a1)

```rust
warning: unreachable `pub` field
 --> src/lib.rs:6:9
  |
6 |         pub b: u8,
  |         ---^^^^^^
  |         |
  |         help: consider restricting its visibility: `pub(crate)`
  |
```

Reply:
That's new to me.
Thanks!

Anyways, amazing library! Was just nitpicking...